graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/FieldTypeMapper.java

```diff
@@ -53,8 +54,9 @@
      */
     private static final ImmutableMap<String, FieldTypes.Type> TYPE_MAP = ImmutableMap.<String, FieldTypes.Type>builder()
             .put("keyword", STRING_TYPE) // since ES 5.x
-            .put("text", STRING_TYPE) // since ES 5.x
-            .put("string", STRING_TYPE) // until ES 2.x
+            .put("text", STRING_FTS_TYPE) // since ES 5.x
+            // TODO: Depending on the value of the "index" field, a "string" can be similar to "keyword" or "text".
+            .put("string", STRING_FTS_TYPE) // until ES 2.x
```

Comment:
With Graylog 3.0.0 supporting only ES 5.x and 6.x, I'm not sure we need to support ES 2.x data types. ES 6.x won't be able to read indices written by ES 2.x anyway.

Reply:
Good point, yes. :+1: I will remove it.